cli/cmd/sync.go

```diff
@@ -65,8 +66,17 @@ func sync(cmd *cobra.Command, args []string) error {
 }
 
 func syncConnection(ctx context.Context, sourceSpec specs.Source, destinationsSpecs []specs.Destination) error {
+	destinationNames := make([]string, len(destinationsSpecs))
+	for i := range destinationsSpecs {
+		destinationNames[i] = destinationsSpecs[i].Name
+	}
+	log.Info().Str("source", sourceSpec.Name).Strs("destinations", destinationNames).Msg("Start sync")
```

Comment:
Do you think it would make sense to set and log `syncTime` here as well? or overwrite the log line timestamp somehow?

Reply:
Oh yeah totally, let's add `syncTime` as that is pretty important.

Reply:
Added